types/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/infobloxopen/protoc-gen-gorm/types
+
+go 1.23.4
+
+require (
+	github.com/golang/protobuf v1.5.4
+	github.com/google/go-cmp v0.6.0
+	github.com/lib/pq v1.10.9
```

Comment:
Looks like this might not actually be needed: https://github.com/infobloxopen/protoc-gen-gorm/blob/7402b542b5ddec9b0d47bdb28ae1d46642c0ef95/types/inet_test.go#L8
I don't see anything in this file that would depend on pq being registered.

Reply:
you are right, I removed and re-ran tests seems fine.